cmd/common/oci.go

```diff
@@ -75,6 +76,20 @@ func NewRunCommand(rootCmd *cobra.Command, runtime runtime.Runtime, hiddenColumn
 			}
 			defer runtime.Close()
 
+			// set global operator flags from the config file
+			for o, p := range opGlobalParams {
+				err = SetFlagsForParams(cmd, p, config.OperatorPrefix+o+".")
```

Comment:
I have the `.` in config.OperatorPrefix so it looks bit off here. Eventually this is something like `operator.localmanager.` and merged with a config key like `runtimes`. Let me see if it can be made more readable!